Rozetka/Mobile.cs

```diff
@@ -4,7 +4,6 @@ public class Mobile: Product
     {
         public Mobile()
```

Comment:
Yes, it can be the case. But now it is not possible to rename this word in all places without  crashes, dependencies, dlls, generated g.cs files.. I tried to do replace this word in all places, but some part of app corrupt, so, I restored the whole project from the github again.